src/components/molecules/ProfileMenu/ProfileMenu.vue

```diff
@@ -56,7 +56,7 @@
       <div class="pl-8 pt-4">
         <a
           href="wishlist.html"
-          class="relative medium capitalize font-medium hover:text-primary transition block text-primary"
+          class="relative medium text-gray-800 capitalize font-medium hover:text-primary transition block "
```

Comment:
tạo một list url rồi dùng v-for để mapping ra, ko hard code như thế này
vd: menuList = [
{
id: 1, 
url: ROUTES.HOME,
label: 'homepage',
icon: h(...)
},
...
]